pkg/operator/sync_controllermanager_v311_00.go

```diff
@@ -341,6 +369,34 @@ func manageServiceCatalogControllerManagerDeployment_v311_00_to_latest(
 	return resourceapply.ApplyDaemonSet(client, recorder, required, resourcemerge.ExpectedDaemonSetGeneration(required, generationStatus), forceRollout)
 }
 
+func addVolumeToDaemonSet(required *appsv1.DaemonSet) {
+	// volumes:
+	//   - name: config-volume
+	//     configMap:
+	//       name: trusted-ca-bundle
+	//       items:
+	//       - key: ca-bundle.crt
+	//         path: "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+
+	required.Spec.Template.Spec.Volumes = append(required.Spec.Template.Spec.Volumes,
+		corev1.Volume{
+			Name: "config-volume",
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "trusted-ca-bundle",
```

Comment:
@jmrodri I'm not sure if `optional: true` is the default for `ConfigMapVolumeSource`, but it might not be a bad idea to be explicit by setting it.